src/iframe-helper.js

```diff
@@ -380,3 +380,61 @@ function parseIfNeeded(data) {
   }
   return data;
 }
+
+
+
+/**
+ * Manages a postMessage API for an iframe with a subscription message and
+ * a way to broadcast messages to all subscribed windows, which
+ * in turn must all be descendants of the contentWindow of the iframe.
+ */
+export class SubscriptionApi {
```

Comment:
Shall we make it a function instead? Seems we don't really need to keep those states. Or, do we plan to introduce other APIs later on?